requirements.txt

```diff
@@ -8,3 +8,5 @@ pytest-cov==3.0.0
 pytest-timeout==2.1.0
 pytest-xdist==2.5.0
 types-setuptools==57.4.18
+semantic_version==2.10.0
+docstring_parser==0.14.1
```

Comment:
@jdtzmn How did you encounter a situation requiring adding these dependencies?

* Take with a grain of salt because my Python package management knowledge is weak:  My understanding is `make setup-development` ought to provide these dependencies from `setup.py`.  Otherwise, I'd expect recent builds in `master` to be failing (https://github.com/algorand/pyteal/actions?query=branch%3Amaster). 
* Locally, I also provisioned a fresh virtual env and successfully bootstrapped via `make make setup-development generate-init`.

Reply:
Yes, that's correct - I must've missed that command. I assumed all the necessary dependencies would be within `requirements.txt` so when I ran tests I was seeing errors about missing those dependencies. I'll delete the commit in which I added these to `requirements.txt`.